.gitignore

```diff
@@ -6,3 +6,6 @@ build*
 *.bak
 src/mlpack/core/util/gitversion.hpp
 src/mlpack/core/util/arma_config.hpp
+
+.idea
```

Comment:
You can also create or modify the global ``~/.gitignore`` file or any other file:

``
git config --global core.excludesfile '~/.gitignore'
``